logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/auth/LoginUseCase.kt

```diff
@@ -106,9 +108,10 @@ internal class LoginUseCaseImpl internal constructor(
     ): AuthenticationResult {
         // remove White Spaces around userIdentifier
         val cleanUserIdentifier = userIdentifier.trim()
-
+        var isEmail = false
```

Comment:
hmmm.. this is a bit weird ? I know that here is not much going on, but I think mutability could bite us when we are doing more stuff here, it makes stuff less predictiable and solving bugs harder, if there is a way to go around this mutability I would say lets try to remove it